src/go/rpk/pkg/cli/cmd/acl/user.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package acl
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/Shopify/sarama"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+)
+
+type NewUser struct {
+	User      string `json:"username"`
+	Password  string `json:"password"`
+	Algorithm string `json:"algorithm"`
+}
+
+const endpoint = "/v1/security/users"
+
+func NewUserCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:          "user",
+		Short:        "Manage users",
+		SilenceUsage: true,
+	}
+
+	command.AddCommand(NewCreateUserCommand())
+	return command
+}
+
+func NewCreateUserCommand() *cobra.Command {
+	var (
+		apiUrl   string
+		user     string
+		password string
+	)
+	command := &cobra.Command{
+		Use:          "create",
+		Short:        "Create users",
+		SilenceUsage: true,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			u := NewUser{
+				User:      user,
+				Password:  password,
+				Algorithm: sarama.SASLTypeSCRAMSHA256,
+			}
+			bs, err := json.Marshal(u)
+			if err != nil {
+				return errors.Wrap(err, "couldn't encode request")
+			}
+			url := fmt.Sprintf("%s%s", apiUrl, endpoint)
+			res, err := http.Post(url, "application/json", bytes.NewBuffer(bs))
+			if err != nil {
+				return err
+			}
+			if res.StatusCode >= 400 {
+				body, err := ioutil.ReadAll(res.Body)
+				log.Debugf("couldn't decode the response body, %v", err)
+				return fmt.Errorf("request to '%s' failed with status %d: %s", apiUrl, res.StatusCode, body)
+			}
+			log.Infof("Created user '%s'.", user)
+			return nil
+		},
+	}
+	command.Flags().StringVar(
+		&apiUrl,
+		"api-url",
+		fmt.Sprintf("http://localhost:%d", config.DefaultAdminPort),
```

Comment:
i think this needs the config too no? 

what about mTLS for the admin api?

Reply:
@senior7515 check the last commit (heads-up: it's still messy). I have reset this as draft, so if you'd like I'll let you know when it's ready for review.